src/Sylius/Behat/Page/Admin/Administrator/UpdatePageInterface.php

```diff
@@ -30,4 +30,8 @@ public function changeLocale(string $localeCode): void;
     public function removeAvatar(): void;
 
     public function hasAvatar(string $avatarPath): bool;
+
+    public function changeTimezone(string $timezone): void;
```

Comment:
I would put this method under `changeLocale`, to keep all `change*` methods together

Reply:
I've changed it too in UpdatePage 😃